README.md

```diff
@@ -24,9 +24,16 @@ This ChIP-Seq pipeline is based off the ENCODE (phase-3) transcription factor an
 
 3) Run a pipeline with Caper.
 
+## DNAnexus
+
+You can also run our pipeline on DNAnexus without using Caper or Cromwell. There are two ways to build a workflow on DNAnexus based on our WDL.
+
+1) [dxWDL CLI](docs/tutorial_dx_cli.md)
+2) [DNAnexus Web UI](docs/tutorial_dx_web.md)
+
 ## Conda
 
-We don't recommend Conda for dependency helper. Use Docker or Singularity instead. We will not take any issues about Conda. You can install Singularity locally without super-user privilege and use it for our pipeline with Caper (with `--use-singularity`).
+We don't recommend Conda for resolving software dependencies. Use Docker or Singularity instead. We will not take any issues about Conda. You can install Singularity and use it for our pipeline with Caper (by adding `--use-singularity` to command line arguments).
```

Comment:
See my comment on https://github.com/ENCODE-DCC/atac-seq-pipeline/pull/128/files#diff-04c6e90faac2675aa89e2176d2eec7d8

Reply:
Fixed in commit f03f851